Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -287,7 +289,16 @@ open class ComposerVC: _ViewController,
 
     /// A Boolean value indicating whether the attachments are enabled.
     open var isAttachmentsEnabled: Bool {
-        channelConfig?.uploadsEnabled == true
+        channelController?.canUploadFile == true
```

Comment:
Is this actually correct? Files are only one type of attachments, does this entry apply to all of them (incl. images, gifs, etc).

Reply:
Yes, at least Android also has this same logic